projects/Izimi/info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Izimi",
+  "description": "Izimi is een digitale kluis aangeboden door het notariaat. Een veilige app om al je gevoelige en belangrijke documenten te bewaren.",
+  "publisher": "In The Pocket",
+  "developers": [],
```

Comment:
You could add a single developer here and leave its Github username empty, but fill in the company. That way In The Pocket is listed as the agency (involved company). I think you might have intended to [put Fednot as the publisher of the app](https://apps.apple.com/be/developer/fednot/id1536437102)?
```
{
  "githubUserName": "",
  "companyName": "In The Pocket"
}

Reply:
Fixed!